bounties/other/NNI/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-19",
+    "AffectedVersionRange": "*",
+    "Summary": "Resources Downloaded over Insecure Protocol",
+    "Contributor": {
+        "Discloser": "b1nslashsh",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "other",
+        "Name": "Microsoft NNI",
+        "URL": "https://github.com/microsoft/nni/",
+        "Downloads": "0"
+    },
+    "CWEs": [
+        {
+            "ID": "494",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
+        "UI": "R",
+        "S": "C",
+        "C": "L",
+        "I": "L",
+        "A": "N",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "4.7"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/microsoft/nni",
+        "Codebase": [
+            "JavaScript"
+        ],
+        "Owner": "Microsoft",
+        "Name": "NNI",
+        "Forks": 950
+        "Stars": 6900
+    },
+    "Permalinks": [
+        "https://github.com/microsoft/nni/blob/0e49e6591a865843035b65dff4fc819e097ab492/examples/trials/weight_sharing/ga_squad/download.sh#L5"
```

Comment:
Change the permalink to the file in the **master** branch:

```
https://github.com/microsoft/nni/blob/master/examples/trials/weight_sharing/ga_squad/download.sh#L5
```

Reply:
Oops my BAD changed  <3